test/importPipeline.js

```diff
@@ -2,23 +2,40 @@ var fs = require( 'fs' );
 var path = require( 'path');
 
 var tape = require( 'tape' );
-var event_stream = require( 'event-stream' );
 var deep = require( 'deep-diff' );
 var map = require('through2-map');
 var _ = require('lodash');
 var proxyquire = require('proxyquire');
-
+const stream = require('stream');
 
 var basePath = path.resolve(__dirname);
 var expectedPath = basePath + '/data/expected.json';
 var inputFiles =  [ basePath + '/data/input_file_1.csv', basePath + '/data/input_file_2.csv'];
 
+function writableStreamArray(cb) {
```

Comment:
Same as in https://github.com/pelias/geonames/pull/366, you can in fact simplify these tests and avoid having to define a custom function.